### DISCUSSION TITLE
How to reset password for admin user ?

### DISCUSSION THREAD

**Discussion by AmadeuAntunes** [OrchardCMS/OrchardCore#8735] [Q&A]
How to reset password for admin user ?

I noticied that password are storage in dbo.Document but how can I set a new password for admin?

**Comment by Skrypt** [comment]
You log in with the admin account and then you go to /admin backend. Left menu : Security --> Users. Then you click on the "Edit password" button.

**Reply by AmadeuAntunes** [reply]
And what if I had lose access to admin backend?

**Reply by Skrypt** [reply]
You can activate the "reset password" feature from the admin website. It will add a "reset password" link to the login page. It will send you an email with a reset password url. Assuming you have used a proper email on the account.

**Reply by hishamco** [reply]
> And what if I had lose access to admin backend?

If you have an access to the database you can replace a secured password with a known one from another app, but that's a trick ;)

**Reply by hishamco** [reply]
What I was talking about when you miss a super admin password

**Reply by hishamco** [reply]
What I was talking about when you miss a super admin password
